testsuite/features/support/cobbler_test.rb

```diff
@@ -57,7 +71,7 @@ def profile_create(name, distro, location)
       @server.call('modify_profile', profile_id, 'distro', distro, @token)
       @server.call('modify_profile', profile_id, 'kickstart', location, @token)
     rescue
-      raise 'modify profile failed.' + $ERROR_INFO.to_s
+      raise 'modifying profile failed.' + $ERROR_INFO.to_s
```

Comment:
OK, apparently we are raising exception in the rescue section in that whole class :)
So, then maybe that's a refactor for another moment... but I would consider refactor and not raise an exception inside a rescue.

Reply:
Agreed. I will open another issue in our board about improving this.

Reply:
https://github.com/SUSE/spacewalk/issues/17382